db/seeds.rb

```diff
@@ -8,3 +8,9 @@
 #
 # Remember to ensure that the commands in this file are idempotent!
 Instance.find_or_create_by(name: 'Default', host: '*')
+
+# Create the default user account.
+user = UserEmail.find_by_email('test@example.org')
+unless user
```

Comment:
NewbZ question: can we use `find_or_create_by`? It does the same thing as what you're testing for here isn't it? Or is there some gotchas?